<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< Updated upstream
    <width>851</width>
    <height>565</height>
=======
    <width>832</width>
    <height>622</height>
>>>>>>> Stashed changes
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
<<<<<<< Updated upstream
   <widget class="QVTKOpenGLWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>571</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraReset">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Camera</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="cubeButton">
         <property name="text">
          <string>Cube</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pyramidButton">
         <property name="text">
          <string>Pyramid</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>162</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="clipFilter">
       <property name="text">
        <string>Clip Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="shrinkFilter">
       <property name="text">
        <string>Shrink Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ObjectColor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Object Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackgroundColor">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Background Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xaxisButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X  plane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xaxisButton2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-X Plane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="YaxisButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y  plane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="YaxisButton2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-Y  plane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xaxisButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X  plane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZaxisButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z  plane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZaxisButton2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-Z  plane</string>
    </property>
   </widget>
=======
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="shrinkFilter">
        <property name="text">
         <string>Shrink Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="filterLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Filters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="clipFilter">
        <property name="text">
         <string>Clip Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="editFilters">
        <property name="text">
         <string>Edit Filters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cameraReset">
        <property name="text">
         <string>Reset Camera</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3" rowspan="2">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Light intensity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="axisLabel">
        <property name="text">
         <string>Axis</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>ON/OFF</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="distWid">
        <property name="text">
         <string>Measurement</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="Slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ObjectColor">
        <property name="text">
         <string>Object Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BackgroundColor">
        <property name="text">
         <string>Background Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QVTKOpenGLWidget" name="qvtkWidget" native="true">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="xaxisButton">
        <property name="text">
         <string>X  plane</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="xaxisButton2">
        <property name="text">
         <string>-X Plane</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="YaxisButton">
        <property name="text">
         <string>Y  plane</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="YaxisButton2">
        <property name="text">
         <string>-Y  plane</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="ZaxisButton">
        <property name="text">
         <string>Z  plane</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="ZaxisButton2">
        <property name="text">
         <string>-Z  plane</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="degreesButton">
        <property name="text">
         <string>90 degrees</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="degreesButton2">
        <property name="text">
         <string>-90 degrees</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>qvtkWidget</zorder>
   <zorder></zorder>
>>>>>>> Stashed changes
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
<<<<<<< Updated upstream
     <width>851</width>
=======
     <width>832</width>
>>>>>>> Stashed changes
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuWidgets">
    <property name="title">
     <string>Widgets</string>
    </property>
    <addaction name="actionBox"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWidgets"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>Icons/fileopen.png</normaloff>
     <normalon>../../../../Downloads/fileopen.png</normalon>Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a STL file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Box Widget</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
